README.md

```diff
@@ -18,5 +17,10 @@ https://community.notepad-plus-plus.org/topic/16566/support-for-plugins-admin-np
 Build Status
 ------------
 
-[![Appveyor build status](https://ci.appveyor.com/api/projects/status/github/notepad-plus-plus/nppPluginList?branch=master&svg=true)](https://ci.appveyor.com/project/donho/nppPluginList)
-[![GitHub release](https://img.shields.io/github/release/notepad-plus-plus/nppPluginList.svg)](https://github.com/notepad-plus-plus/nppPluginList/releases)
+[![Appveyor build status][Appveyor-build-img]][Appveyor-build-url]
+[![GitHub release][Gitlab-img]][Gitlab-url]
```

Comment:
Wrong naming with gitlab.
What is the advantages to have this splitted up as they are just use once?